src/sentry/static/sentry/app/views/eventsV2/utils.tsx

```diff
@@ -353,6 +364,24 @@ function generateExpandedConditions(
   // Add additional conditions provided and generated.
   for (const key in conditions) {
     const value = conditions[key];
+
+    if (Array.isArray(value)) {
+      parsedQuery.addOp('(');
+
+      const lastIndex = value.length - 1;
+
+      value.forEach((tagValue, index) => {
+        parsedQuery.addTagValues(key, [tagValue]);
+
+        if (lastIndex !== index) {
+          parsedQuery.addOp('OR');
```

Comment:
I know we don't have array comparisons yet but let's use an `AND` here to be closer to array equality.